shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -5,6 +5,7 @@ import com.kcc.kmmhackathon.shared.entity.MealWithDistance
 import com.kcc.kmmhackathon.shared.entity.Quantity
 import com.kcc.kmmhackathon.shared.utility.DistanceUnit
 import com.kcc.kmmhackathon.shared.utility.LocationUtil
+import com.kcc.kmmhackathon.shared.utility.DateKMM
```

Comment:
Had to call it DateKMM because Xcode gets itself all confused when I called it Date - which is to be expected, feel free to suggest a better name 🤷

Reply:
You can disambiguate by including the module name before when using Date like `Foundation.Date` or `Swift.Date` (can't remember which) and `shared.Date` but that might get tedious, so makes sense to typealias. Not super keen on the name, but it works. Maybe something like SharedDate, XDate (as in, cross platform date)

Reply:
I might go with sharedDate 👍